docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+services:
+  dogecoind:
+    container_name: dogecoind_regtest_bob
+    image: dogecoind_bob:latest
+    # specify additional command line arguments here
+    command: dogecoind, dogecoin-tx, test_dogecoin
+    healthcheck:
+      test: ["CMD", "/entrypoint.sh", "dogecoin-cli", "getpeerinfo"]
```

Comment:
We could use `ping` instead of `getpeerinfo`

Reply:
sure that sgtm.